src/wallet/rpcwallet.cpp

```diff
@@ -4773,6 +4773,31 @@ size_t EstimateTxSize(
     return txsize;
 }
 
+static std::optional<Memo> ParseMemo(const UniValue& memoValue)
```

Comment:
Should `z_shieldcoinbase` also use it?

Reply:
`z_shieldcoinbase` doesn’t yet accept a `memo` at all. I think it should … if we get it in this release 😁, we could put it in the ”proper” position ahead of `privacyPolicy`, otherwise I assume it’ll have to go at the end (after `privacyPolicy`, and not match `z_mergetoaddress`.

Reply:
Hm, yeah, if it's not a huge lift, another commit on here that adds the memo parameter to `z_shieldcoinbase` would be nice, so that the order of parameters is consistent.

Reply:
Yeah, I think it’s trivial, but I’ll do a separate PR.

Reply:
#6604 (based on this PR)